vault/logical_system_helpers.go

```diff
@@ -139,3 +139,7 @@ func (b *SystemBackend) tuneMountTTLs(ctx context.Context, path string, me *Moun
 
 	return nil
 }
+
+func (b *SystemBackend) wrapHandleRaftRemovePeer() framework.OperationFunc {
+	return b.handleRaftRemovePeerUpdate()
+}
```

Comment:
Checkout the var definitions at the top of this file, i think you'll want to do something similar so you can override the function on ent.